tgui/packages/tgui/styles/interfaces/KitchenMachine.scss

```diff
@@ -0,0 +1,14 @@
+@use '../colors.scss';
+
+.Ingredient__Table {
+  tr:nth-child(even) {
+    background-color: #333
+  }
+}
+
+table {
+  border-spacing: 10px;
+}
+td {
+  padding: 5px;
+}
```

Comment:
these need to be moved inside the `.table` class styling block. Otherwise you're applying this styling to ALL tables.

Reply:
![image](https://user-images.githubusercontent.com/91113370/193991325-03d91b3a-fd7f-4766-82e5-a97bd5d10e26.png)
![image](https://user-images.githubusercontent.com/91113370/193991348-7f791d2a-7261-4e45-bf69-630086932fb5.png)
Tried doing this but it annhilates it, but I gotta sleep for now. Pointers on this appreciated.

Reply:
Nevermind, found out how to do it